client/components/SongBarEntry.jsx

```diff
@@ -1,16 +1,21 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const SongBarEntry = ({ height, bar }) => (
-  <div
-    className="player-songbar-bar"
-    style={{ height: height * bar }}
-  />
-);
+const SongBarEntry = ({ position, bar, currentTime, totalTime }) => {
+  const height = (position === 'lower') ? 20 : 80;
```

Comment:
the prop types currentTime and Total Time isn't being used currently - if you don't need them, i would suggest taking them out unless you're waiting to use them later?

Reply:
I actually refactored this is my other branch (songbar_dev). They are now contained in a container object!